mobile/assets/i18n/en-US.json

```diff
@@ -226,28 +240,23 @@
   "login_form_server_error": "Could not connect to server.",
   "login_password_changed_error": "There was an error updating your password",
   "login_password_changed_success": "Password updated successfully",
-  "map_assets_in_bounds": {
-    "one": "{} photo",
-    "other": "{} photos"
-  },
+  "map_assets_in_bounds[plural_one]": "{} photo",
```

Comment:
Are these from localizely or are these manually edited with the new key names? Asking this because these are still wrong and would still be an issue. The actual usage of these keys still refer to the now removed key name. For example, here is the one for `map_assets_in_bounds`:

https://github.com/immich-app/immich/blob/65402026528529d7dd44a64e6fc3aedda4c9e453/mobile/lib/modules/map/ui/map_page_bottom_sheet.dart#L179-L180

Reply:
Those are from Localizely